olcut-config-edn/src/test/java/com/oracle/labs/mlrg/olcut/config/edn/test/AllFieldsConfiguredTest.java

```diff
@@ -68,14 +69,14 @@ public void setUp() throws IOException {
 
     @Test
     public void loadConfig() throws IOException {
-        ConfigurationManager cm = new ConfigurationManager("allConfig.edn");
+        ConfigurationManager cm = new ConfigurationManager(this.getClass().getName()+"|allConfig.edn");
```

Comment:
That sounds like a good idea. I'd lean towards `OLCUTResourceName` and try to finesse it a little so that it works without modules too, though I'm not sure if we're going to really support that for much longer.

Reply:
Ended up going with a static method after our discussion.